torch_geometric/transforms/add_metapaths.py

```diff
@@ -87,15 +87,20 @@ class AddMetaPaths(BaseTransform):
         max_sample (int, optional): If set, will sample at maximum
             :obj:`max_sample` neighbors within metapaths. Useful in order to
             tackle very dense metapath edges. (default: :obj:`None`)
+        weighted (bool, optional): If set to :obj:`True` compute weights for
+            each metapath and store them in :obj:`edge_attr`.
+            (default :obj:`False)
+        use_edge_attr_as_weights (bool, optional): If set to :obj:`True`
```

Comment:
is this variable only useful conditioned on the previous argument? Maybe we could document a bit more about the relations with the weighted argument

Reply:
Yes.  Clarified.